app/assets/javascripts/client/inventories/InventoryDeadlineModalCtrl.js

```diff
@@ -0,0 +1,36 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .controller('InventoryDeadlineModalCtrl', InventoryDeadlineModalCtrl);
+
+  InventoryDeadlineModalCtrl.$inject = [
+    '$scope',
+    '$timeout',
+    'Inventory'
+  ];
+
+  function InventoryDeadlineModalCtrl($scope, $timeout, Inventory) {
+    var vm = this;
+    vm.inventory = $scope.inventory;
+    vm.closeModal = function() {
+      $scope.$emit('close-deadline-modal');
+    }
+
+    $timeout(function() {
+      if(vm.inventory.due_date != null) {
+        $scope.modal_due_date = moment($scope.inventory.due_date).format("MM/DD/YYYY");
+      }
+
+      $('.datetime').datetimepicker({pickTime: false});
```

Comment:
This looks fine to me and we've done this sort of thing before.  In the future though, I'd like to be able to push this down a layer to a lifecycle component that guarantees the existence of the DOM before we have to do anything in the controller; probably through a `link` function.